pkgs/ffigen/pubspec.yaml

```diff
@@ -17,21 +17,26 @@ environment:
   sdk: '>=3.3.0-252.0.dev <4.0.0'
 
 dependencies:
-  ffi: ^2.0.1
-  yaml: ^3.0.0
-  path: ^1.8.0
-  quiver: ^3.0.0
   args: ^2.0.0
-  logging: ^1.0.0
   cli_util: ^0.4.0
-  glob: ^2.0.0
+  collection: ^1.18.0
+  ffi: ^2.0.1
   file: ^7.0.0
+  glob: ^2.0.0
+  logging: ^1.0.0
   package_config: ^2.1.0
+  path: ^1.8.0
+  objective_c: ^0.0.1
```

Comment:
Should not be a dependency at all of ffigen, but only a dependency of the test projects?

Reply:
Done